<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyNote</class>
 <widget class="QMainWindow" name="EasyNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyNote</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="contentTextEdit">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="saveFileAsAction"/>
    <addaction name="separator"/>
    <addaction name="pageSetAction"/>
    <addaction name="printAction"/>
    <addaction name="separator"/>
    <addaction name="quitAciton"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_U_2"/>
    <addaction name="separator"/>
    <addaction name="action_T"/>
    <addaction name="action_C"/>
    <addaction name="action_P_2"/>
    <addaction name="action_L"/>
    <addaction name="separator"/>
    <addaction name="action_F"/>
    <addaction name="action_N"/>
    <addaction name="action_R"/>
    <addaction name="action_G"/>
    <addaction name="separator"/>
    <addaction name="action_A_2"/>
    <addaction name="action_D"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="action_W"/>
    <addaction name="action_F_2"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="action_S_2"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="separator"/>
    <addaction name="action_A_3"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newFileAction"/>
   <addaction name="openFileAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFileAction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/新建.png</normaloff>:/icons/新建.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N）</string>
   </property>
   <property name="toolTip">
    <string>创建一个文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openFileAction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/图片.png</normaloff>:/icons/图片.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveFileAsAction">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="pageSetAction">
   <property name="text">
    <string>页面设置(&amp;U)</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="quitAciton">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_U_2">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_P_2">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
  </action>
  <action name="action_N">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>转到(&amp;G)</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>日期/时间(&amp;D)</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="action_F_2">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="action_S_2">
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="action_A_3">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
